pkg/cmd/factory/http.go

```diff
@@ -127,6 +128,22 @@ func NewHTTPClient(io *iostreams.IOStreams, cfg configGetter, appVersion string,
 	return api.NewHTTPClient(opts...), nil
 }
 
+var ssoHeader string
```

Comment:
Keeping this state around from the request feels a bit off. I am wondering if we could actually do something similar to what we do with scope suggestions and modify the http response error message to indicate that it was a SSO auth challenge error and what the SSO URL is?

Reply:
Sure, it's not the most elegant solution, but the `shurcool-graphql` library does not allow access to HTTP response headers once it has parsed the GraphQL error information. Additionally, its `errors` type is an unexported type. Thus, we cannot decorate its error object with this information.

Reply:
That makes sense. The current approach seems appropriate then.